### PR TITLE
Show grants worth when estimation can be made

### PR DESCRIPTION
**Pull request by sh0oki**

### CODE REVIEW

**Comment by rootux** [comment]
Looks good.
I'm missing - Documenting all those things in readme.
**Granting**
**Step 1 - Granting starts**
Set the `lockdown.yml` vote_hearts - we will use this value to automatically close the voting.
Set under `application.rb` the `budget_value` - we will use it for Flow
**Step 2 - Granting ends**
We will use from `application.rb` the `budget_value` - to estimated the amount the each dream was granted.

Also locally I got some weird results - perhaps cause the total sum was less then 50,000 nis.

screen shot 2017-04-23 at 2 18 41 pm

Later when I changed to 1000 total nis I got the next image who looks better
screen shot 2017-04-23 at 2 22 41 pm

**Comment by rootux** [comment]
But still I would love to know how many 'hearts' they got and the numbers you are showing there under estimation doesn't do that. perhaps in a small text next to it (To show people the raw data we've used for calculation)

File: app/views/camps/_list.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
                 <div class="panel panel-info">
                     <div class="panel-body camp-index">
                         <% if camp.images.any? %>
-                            <%= image_tag(camp.images.first.attachment.url(:square), :class => "grow") %>
+                            <%= image_tag("https://s3-eu-central-1.amazonaws.com/dreams.midburn.org" + camp.images.first.attachment.url(:square)[7..-1], :class => "grow") %>
```
Comment (rootux) [inline app/views/camps/_list.html.erb]: What is this? Have you added showing the first attached image? @sh0oki

File: app/views/camps/show.html.erb [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
         <% content_for(:post_head) do %>
             <% if defined? (@mainImage) %>
-                <div class='dream_main_image' style='background: linear-gradient(rgba(20,20,20, .7),rgba(20,20,20, .7)),url("<%= @mainImage %>") no-repeat center center fixed'>
+                <div class='dream_main_image' style='background: linear-gradient(rgba(20,20,20, .7),rgba(20,20,20, .7)),url("https://s3-eu-central-1.amazonaws.com/dreams.midburn.org<%= @mainImage[7..-1] %>") no-repeat center center fixed'>
```
Comment (rootux) [inline app/views/camps/show.html.erb]: What does 7..-1 mean?

**Comment by sh0oki** [comment]
Please ignore both of these (s3 link and [-7...]), i did it to see the s3
images on my localhost debuggings...

**Pull request merged**